index.html

```diff
@@ -0,0 +1,217 @@
+            <h3 class="project-title">Tonic</h3>
+            <div class="explaining-spans">
+              <span>CANOPY</span>
+              <span></span>
+              <span>Back end dev</span>
+              <span></span>
+              <span>2015</span>
+            </div>
+            <p class="short-description">
+              A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+            <div class="badges-wrapper">
+              <span class="badge">HTML</span><span class="badge">CSS</span><span class="badge">JavaScript</span>
+            </div>
+            <button class="buttons" type="button">See Project</button>
+          </article>
+          <article class="card">
+            <figure>
+              <img src="/assets/images/mobile/Snapshoot Portfolio-1.png" alt="" class="project-img">
+            </figure>
+            <h3 class="project-title">Multi-Post Stories</h3>
+            <div class="explaining-spans">
+              <span>CANOPY</span>
+              <span></span>
+              <span>Back end dev</span>
+              <span></span>
+              <span>2015</span>
+            </div>
+            <p class="short-description">
+              A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+            <div class="badges-wrapper">
+              <span class="badge">HTML</span><span class="badge">CSS</span><span class="badge">JavaScript</span>
+            </div>
+            <button class="buttons" type="button">See Project</button>
+          </article>
+          <article class="card">
+            <figure>
+              <img src="/assets//images//mobile//Snapshoot Portfolio-2.png" alt="" class="project-img">
+            </figure>
+            <h3 class="project-title">Tonic</h3>
+            <div class="explaining-spans">
+              <span>CANOPY</span>
+              <span></span>
+              <span>Back end dev</span>
+              <span></span>
+              <span>2015</span>
+            </div>
+            <p class="short-description">
+              A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+            <div class="badges-wrapper">
+              <span class="badge">HTML</span><span class="badge">CSS</span><span class="badge">JavaScript</span>
+            </div>
+            <button class="buttons" type="button">See Project</button>
+          </article>
+          <article class="card">
+            <figure>
+              <img src="/assets/images/mobile/Snapshoot Portfolio-3.png" alt="" class="project-img">
+            </figure>
+            <h3 class="project-title">Multi-Post Stories</h3>
+            <div class="explaining-spans">
+              <span>CANOPY</span>
+              <span></span>
+              <span>Back end dev</span>
+              <span></span>
+              <span>2015</span>
+            </div>
```

Comment:
Instead of using span here, I think it will be better to make use of `li` wrapped in the `ul` tag since we are working with lists here. 😎

Reply:
- [ OPTIONAL]AS required from the previous review you should use `li` wrapped in the `u;` tag as you are working with lists :wink: